modules/govuk_ci/manifests/agent.pp

```diff
@@ -31,16 +31,21 @@
   include ::govuk_ci::agent::rabbitmq
   include ::govuk_ci::credentials
   include ::govuk_ci::limits
-  if $licensing_tests {
-    include ::govuk_java::oracle8
-  }
   include ::govuk_jenkins::packages::terraform
   include ::govuk_jenkins::pipeline
   include ::govuk_jenkins::user
   include ::govuk_rbenv::all
   include ::govuk_sysdig
   include ::govuk_testing_tools
 
+  if $licensing_tests {
+    include ::govuk_java::oracle8
+  } else {
+    package { 'openjdk-7-jdk':
+      ensure => 'present',
```

Comment:
Should this include `govuk_java::openjdk7::jdk` instead?

Reply:
Oh, probably :)